doc/src/tutorials/biomechanics/biomechanical-model-example.rst

```diff
@@ -4,8 +4,8 @@
 Biomechanical Model Example
 ===========================
 
-:obj:`~sympy.physics.biomechanics` provides features to enhance models created
-with :obj:`~sympy.physics.mechanics` with force producing elements that model
+:obj:`sympy.physics.biomechanics` provides features to enhance models created
+with :obj:`sympy.physics.mechanics` with force producing elements that model
```

Comment:
The purpose of the tilde is supposed to be that only "biomechanics" will be displayed, instead of the full path "sympy.physics.biomechanics", at least that is what I thought.

Reply:
It is. I removed the tildes at the first usages because I thought it gave clearer context, but I can add them back in.

Reply:
No that's fine. The commit message said "fixed incorrect..." so I was wondering why it might be incorrect.